os/arch/arm/src/common/up_createstack.c

```diff
@@ -243,6 +253,16 @@ int up_create_stack(FAR struct tcb_s *tcb, size_t stack_size, uint8_t ttype)
 		heapinfo_exclude_stacksize(tcb->stack_alloc_ptr);
 #endif
 		board_led_on(LED_STACKCREATED);
+#ifdef CONFIG_MPU_STACKGUARD
+		/* The smallest size that can be programmed for an MPU region is 32 bytes */
+		tcb->stack_guard = (uint32_t)tcb->stack_alloc_ptr + 0x40;
```

Comment:
Could you let me know the meaning of `0x40`? Could you use the definition to know what it is easily instead of the value?

Reply:
This is for the memory address alignment needed for MPU region. I have modified with macro now. Reserved stack allocation with 32 byte align. Adding the minimum size required for mpu allocation now (32 bytes).